File: backend/model/petugasLain.go
package model

type PetugasLain struct {
	Id      uint   `json:"id" gorm:"primaryKey"`
	Nama    string `json:"nama" gorm:"not null"`
	NIP     string `json:"nip" gorm:"not null"`
	Pangkat string `json:"pangkat" gorm:"not null"`
	Jabatan string `json:"jabatan" gorm:"not null"`
	Unit    string `json:"unit" gorm:"not null"`
}

func (PetugasLain) TableName() string {
	return "petugas_lain"
}

File: backend/app/Models/Petugas_Lain.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Petugas_Lain extends Model
{
     use HasFactory;

    protected $table = 'pegawai';

    protected $fillable = [
        'nama',
        'nip',
        'jabatan',
        'pangkat',
        'unit',
    ];

    /**
     * Relasi One-to-Many ke Lpd
     */
    public function lpd()
    {
        return $this->hasMany(Lpd::class, 'petugas_lain_id');
    }
}
